google-cloud-dialogflow/Gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+gemspec
+
+gem "gcloud-jsondoc",
+    git: "https://github.com/GoogleCloudPlatform/google-cloud-ruby.git",
+    branch: "gcloud-jsondoc"
+
```

Comment:
You might consider the jwt workaround as well. CI doesn't always fail on Ruby 2.0 without it, but it is possible.